test/qa-tests/jstests/top/mongotop_json.js

```diff
@@ -27,17 +27,17 @@ var testName = 'mongotop_json';
     }, 'invalid JSON 1');
 
     // ensure tool runs without error with --rowcount > 1
-    var rowcount = 5;
+    var rowcount = 8;
```

Comment:
why'd this one jump to 8?

Reply:
Oh, this is a subjective number, currently, it should work with any value after the below change. I will change it back.